libexec/validate-tables.py

```diff
@@ -439,7 +439,9 @@ def check_table(node):
 # Validate DocBook file
 def check_file(filename):
     """Validate tables in DocBook file."""
-    xml = etree.parse(filename)
+    parser = etree.XMLParser(load_dtd=True, huge_tree=True)
+    xml = etree.parse(filename, parser)
+    xml.xinclude()
```

Comment:
I think, you should handle any problems that might occur during parsing or XInclude resolution, otherwise the user gets some ugly tracebacks:

```suggestion
    parser = etree.XMLParser(load_dtd=True, huge_tree=True)
    try:
        xml = etree.parse(filename, parser)
        xml.xinclude()
    except (etree.XIncludeError, etree.XMLSyntaxError) as error:
        print(error, file=sys.stderr)
        sys.exit(10)
```

Reply:
Meh -- you made this a suggestion and ended it with a comment `# handle the cases`. I would have preferred not to have to think. :)

Reply:
Haha, okay, I've amended the part so you can use it without thinking. :wink: 

Yes, it '*shouldn't* be necessary. But it's just a small addition and it improves the usability a lot. Also better safe than sorry.